### PR TITLE
set encoding `utf-8` for the reader and the writer of the file `comments.json`

### PR DESCRIPTION
**Pull request by yepzy**
The web page will had some problem of encoding of the `'`, with the server in python so, I have corrected it, by adding `encoding="utf-8"`when the files comments.json is opened

### CODE REVIEW

File: server.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 # FACEBOOK BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN
 # ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
 # WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
-
+# coding: utf8
```
Comment (zpao) [inline server.py]: I don't think this is necessary

File: server.py [hunk 2]
```diff
@@ -19,15 +19,15 @@
 
 @app.route('/api/comments', methods=['GET', 'POST'])
 def comments_handler():
-    with open('comments.json', 'r') as f:
+    with open('comments.json', 'r',encoding='utf-8') as f:
```
Comment (zpao) [inline server.py]: This and below are fine but please add a space after the comma (for pep8 style) and also please remove all of the code related to your implementation.

**Comment by InceptionCode** [comment]
I think you have to give me access in order to merge the request.

**Comment by zpao** [comment]
@InceptionCode I believe you may be commenting on the wrong repo.

**Comment by InceptionCode** [comment]
@zpao Your right I apologize my repo has the same name.

**Comment by zpao** [comment]
No worries :)

**Pull request closed without merging**